docs/index.md

```diff
@@ -42,7 +42,7 @@ Because the command execution needs root permission, "sudo" is required before "
     npm install -g node-red-nodegen
 
 The current version of Node generator supports function node and Open API document as source files.
-Simply, Node generator command, node-red-nodegen converts to a node from the file which is specified in the argument of the command as follows.
+The Node generator command, node-red-nodegen creates a node from the file which is specified in the argument of the command as follows.
```

Comment:
`The Node generator command, node-red-nodegen creates a node` -> `node-red-nodegen creates a node`